sync/downstream.py

```diff
@@ -758,7 +759,11 @@ def affected_tests(self, revish=None):
             if revish:
                 args.append(revish)
             logger.info("Getting a list of tests affected by changes.")
-            output = self.wpt.tests_affected(*args)
+            try:
+                output = self.wpt.tests_affected(*args)
+            except subprocess.CalledProcessError:
+                logger.error("Calling wpt tests-affected failed")
+                return tests_by_type
```

Comment:
Should we not be setting `self.data["affected-tests"]` like we did before? or are we ignoring this attempt all together if it fails?

Reply:
I think it's OK to just ignore it if it fails.